wgpu/src/backend/web.rs

```diff
@@ -2066,10 +2066,14 @@ impl crate::Context for Context {
         }
 
         if let Some(dsa) = &desc.depth_stencil_attachment {
+            let mut clear_value = 1.0;
```

Comment:
It doesn't matter, but given that the default value is `0.0` we might as well use that.

```suggestion
            let mut clear_value = 0.0;
```

Could even set it to `None` by default and only call `mapped_depth_stencil_attachment.depth_clear_value` if it's `Some`, but I don't think that's necessary.

Reply:
Make definitely sense - default is 0.0